analyzer/compliance-analyzer/src/main/java/eu/fasten/analyzer/complianceanalyzer/Main.java

```diff
@@ -0,0 +1,95 @@
+package eu.fasten.analyzer.complianceanalyzer;
+
+import org.json.JSONObject;
+import org.json.JSONTokener;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+
+
+@CommandLine.Command(name = "ComplianceAnalyzer")
+public class Main implements Runnable {
+
+    private static final Logger logger = LoggerFactory.getLogger(Main.class);
+
+    @CommandLine.Option(names = {"-c", "--credentials", "--cluster-credentials"},
+            paramLabel = "credentials",
+            description = "Path to the Kubernetes cluster credentials file")
+    String clusterCredentialsFilePath;
+
+    // TODO This plugin would consume a Kafka Topic to retrieve repository information
```

Comment:
`Main` is usually used for local testing. It should not use Kafka, as such inputs are provided by FASTEN Server. Other plugins' Main classes take as the input either a path to a file that contains a JSON with the required information (same information which would be supplied by the server from Kafka) or uses the command-line arguments straight away.
For example, RepoCloner can take `-f path/to/file.json` that contains a JSON with `artifactId`, `groupId`, `version`, `repoUrl` and etc, and these parameters can be passed as CLI arguments with `-g <groupId> -a <artifactId> -v <version> -u <repo URL> ...`
Please adjust your `Main` class accordingly, you can see how other plugins' Main classes look like.

Reply:
Ok, now I got how the workflow actually looks like. In 17c7a707 I removed all the comments relative to the consumption of the Kafka Topic in the `Main` class, as it is not supposed to happen. I left everything as it is as the plugin already has a `-r` flag which allows the caller to specify the path to the JSON file containing repository information.